File: src/main/java/org/taktik/icure/dto/filter/patient/PatientByHcPartyNameFilter.java
package org.taktik.icure.dto.filter.patient;

import org.taktik.icure.dto.filter.Filter;
import org.taktik.icure.entities.Patient;

public interface PatientByHcPartyNameFilter extends Filter<String, Patient> {
    String getName();
    String getHealthcarePartyId();
}

File: src/main/kotlin/org/taktik/icure/domain/filter/patient/PatientByHcPartyNameFilter.kt
package org.taktik.icure.domain.filter.patient

import org.taktik.icure.domain.filter.Filter
import org.taktik.icure.entities.Patient

interface PatientByHcPartyNameFilter : Filter<String, Patient> {
    val name: String?
    val healthcarePartyId: String?
}
